wiki/Community/osu!monthly/de.md

```diff
@@ -1,46 +1,37 @@
 
-Mit dem Wunsch, das Projekt zurückzubringen, wurde das [osu!monthly-Team](/wiki/People/osu!monthly_team) mit ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) an der Spitze gebildet, welches das Projekt in Form von Newsbeiträgen wieder zum Leben erweckt hat. Die Serie umfasste Ausgaben für den Zeitraum zwischen Juli 2022 und März 2023.
+## Ursprüngliche Auflage (2011)
 
-## Archiv
+Die ursprüngliche, von ::{ flag=US }:: [jjrocks](https://osu.ppy.sh/users/30985) und ::{ flag=GB }:: [Corin](https://osu.ppy.sh/users/639270) erstellte Auflage von osu!monthly lief von März bis Oktober 2011. Das Projekt begann als Google-Dokument, aber entwickelte sich schnell in ein PDF-Format, das einem gedruckten Magazin ähnelt. Diese Ausgabe war bekannt für ihr großes Spektrum an Themen sowie ihre unverwechselbare und künstlerische Gestaltung.
 
-### Season 1 (2011)
+Weitere Informationen über die erste Version von osu!monthly sind auf der [archivierten Webseite von jjrocks](https://web.archive.org/web/20201112030400/http://jjwanda.com/projects/osumonthly.html) zu finden. Diverse Links zum Projekt sind unten auf der [osu!monthly-Webseite](https://osumonthly.com/) aufgeführt.
 
 | Name | Veröffentlichungsdatum |
 | :-- | :-- |
-| [osu!monthly #1](https://osumonthly.com/pdf/osu!monthly-1.pdf) | 05.03.2011 |
-| [osu!monthly #2](https://osumonthly.com/pdf/osu!monthly-2.pdf) | 06.05.2011 |
-| [osu!monthly #3](https://osumonthly.com/pdf/osu!monthly-3.pdf) | 07.06.2011 |
-| [osu!monthly #4](https://osumonthly.com/pdf/osu!monthly-4.pdf) | 07.07.2011 |
-| [osu!monthly #5](https://osumonthly.com/pdf/osu!monthly-5.pdf) | 15.08.2011 |
-| [osu!monthly #6](https://osumonthly.com/pdf/osu!monthly-6.pdf) | 27.10.2011 |
+| [The osu! Monthly 1st Edition](https://osumonthly.com/pdf/osu!monthly-1.pdf) | 05.03.2011 |
+| [The osu! Monthly May 2011 Edition](https://osumonthly.com/pdf/osu!monthly-2.pdf) | 06.05.2011 |
+| [The osu!monthly June 2011 Edition](https://osumonthly.com/pdf/osu!monthly-3.pdf) | 07.06.2011 |
+| [osu!monthly July 2011 Edition](https://osumonthly.com/pdf/osu!monthly-4.pdf) | 07.07.2011 |
+| [osu!monthly August 2011 Edition](https://osumonthly.com/pdf/osu!monthly-5.pdf) | 15.08.2011 |
+| [osu!monthly October 2011 Edition](https://osumonthly.com/pdf/osu!monthly-6.pdf) | 27.10.2011 |
 
-Weitere Informationen über die erste Version von osu!monthly sind auf der [Webseite von JJ](https://web.archive.org/web/20201112030400/http://jjwanda.com/projects/osumonthly.html) zu finden.
+## Neuauflage des Magazins (2017 bis 2018)
 
-### Season 2 (2017 bis 2018)
+Inspiriert von der ersten Version von osu!monthly, wurde dank ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) sowie ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) ein neues Team gebildet, mit dem Ziel, das Projekt fortzuführen. Von Oktober 2017 bis Februar 2018 wurden neue Editionen des Magazins veröffentlicht.
```

Comment:
I think this wording can be a bit simpler without losing information:
```suggestion
Inspiriert von der ersten Auflage haben ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) sowie ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) ein neues Team gebildet mit dem Ziel, das Projekt fortzuführen. Von Oktober 2017 bis Februar 2018 wurden neue Editionen des Magazins veröffentlicht.
```

Reply:
At first, I thought removing the second comma is a mistake. [Turns out, it is not](https://www.korrekturen.de/forum.pl/md/read/id/107873/sbj/komma-vor-mit-dem-ziel/).

Still sounds rough to me, but eh 🤷